packages/clay-management-toolbar/src/ClayResultsBar.soy

```diff
@@ -0,0 +1,144 @@
+{namespace ClayResultsBar}
+
+/**
+ * This renders the component's whole content.
+ */
+{template .render}
+    {@param spritemap: string}
+	{@param totalItems: number}
+	{@param? _handleClearResultsClick: any}
+    {@param? _handleFilterLabelCloseClicked: any}
+	{@param? clearResultsURL: string}
+	{@param? filterLabels: list<[
+		closeable: bool,
+        label: string
+	]>}
+	{@param? searchValue: string}
+
+	<nav class="tbar subnav-tbar subnav-tbar-primary">
+		<div class="container-fluid container-fluid-max-xl">
+			<ul class="tbar-nav">
+                {call .results}
+                    {param expand: not ($filterLabels and length($filterLabels) > 0) /}
+                    {param searchValue: $searchValue /}
+                    {param totalItems: $totalItems /}
+                {/call}
+            
+				{if $filterLabels}
+                    {call .filterLabels}
+                        {param _handleFilterLabelCloseClicked: $_handleFilterLabelCloseClicked /}	
+                        {param filterLabels: $filterLabels /}	
+                        {param spritemap: $spritemap /}	
+                    {/call}
+				{/if}
+
+                {call .clearResults}
+                    {param _handleClearResultsClick: $_handleClearResultsClick /}
+	                {param clearResultsURL: $clearResultsURL /}
+                {/call}
+			</ul>
+		</div>
+	</nav>
+{/template}
+
+/**
+ * This renders de clear results button or link
+ */
+{template .clearResults}
```

Comment:
I think we should make this a `deltemplate` so this can make it an extension point so people can cover cases like this:

<img width="238" alt="screen shot 2018-08-30 at 21 20 24" src="https://user-images.githubusercontent.com/13750819/44886314-fda29880-ac9b-11e8-8847-8f0d1cb502cd.png">

Maybe rename it to something more general, something like **ResultsAction**.

Reply:
If this is a need please open an issue and we can add it later, I prefer do not mix issues here :)